lib/Controller/PageController.php

```diff
@@ -0,0 +1,23 @@
+<?php

```

Comment:
this class should replace with controller/pagecontroller

Reply:
contorller/pagecontroller.php has moved into lib/Controller/PageController.php